swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, condition = 0, fabric = "Unknown"):
```

Comment:
Based on PEP8 style guide, let's remove the spaces between the `=` in keyword parameters. Consider this feedback for the other class methods in the other files below!
```python
    def __init__(self, id=None, condition=0, fabric="Unknown"):
```
[PEP8 Styling Guide Source](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements:~:text=Don%E2%80%99t%20use%20spaces,%3D%20imag)